LottoDiary/LottoDiary/Sources/Present/ViewController.swift

```diff
@@ -18,6 +18,14 @@ class ViewController: UIViewController {
         v.layer.borderColor = UIColor.designSystem(.mainOrange)?.cgColor
         return v
     }()
+    
+    let codeReviewLabel: UILabel = {
+        let v = UILabel()
+        v.text = "코드리뷰설명 PR을 위한 라벨입니다"
+        v.font = .systemFont(ofSize: 15, weight: .bold)
```

Comment:
```suggestion
        v.font = .systemFont(ofSize: 17, weight: .bold)
```
글자 크기를 키우는게 좋을 것 같습니다.

Reply:
넵